pallets/pallet-bonded-coins/src/lib.rs

```diff
@@ -269,6 +269,11 @@ pub mod pallet {
 		InvalidInput,
 		Internal,
 		Slippage,
+		/// The requested Operation could not be performed due to insufficient
+		/// balance.
+		InsufficientBalance,
```

Comment:
I think that error is misleading, can we call this `BelowMinimum` and describe it as "The amount to mint or burn is below the minimum allowed?"

Reply:
Yeah, naming… Always difficult :grimacing: 

renamed in: [0e603c8](https://github.com/KILTprotocol/kilt-node/pull/803/commits/0e603c81763c2a2c67073a19887099e2be3c9167)